ledger/participant-integration-api/src/main/scala/platform/store/dao/events/EventsTableH2Database.scala

```diff
@@ -171,15 +172,16 @@ object EventsTableH2Database extends EventsTable {
   private val updateArchived =
     """update participant_events set create_consumed_at={consumed_at} where contract_id={contract_id} and create_argument is not null"""
```

Comment:
```suggestion
```

We shouldn't need that anymore should we?

Reply:
Correct.
Please note that building H2 support was not a goal of these changes.
I only did the minimal corrections to make it compile: which also means all test over H2 fail currently.
There is already a TODO to propagate support of the append only schema to H2 artifacts.